.github/workflows/ci.yml

```diff
@@ -34,7 +34,5 @@ jobs:
       - run: yarn install --frozen-lockfile
       - run: make bootstrap
       - run: make packages-build
-      - run: make fmt-check
```

Comment:
`eslint-config-prettier` purpose is:

> Turns off all rules that are unnecessary or might conflict with Prettier.

So I think we still need to run `fmt-check` which serves a different purpose.

Reply:
Are you okay adding eslint-plugin-prettier? I think I meant to do that but missed it.

Reply:
Oh, I wasn't aware of this plugin, I think it's good to be able to run everything with one command, I just hope we'll understand clearly when it's just about re-running prettier VS issues which have to be fixed manually. Let's add it and see how it goes.